packages/market-service/src/coingecko/coingecko.ts

```diff
@@ -37,69 +36,68 @@ type CoinGeckoAssetData = {
   }
 }
 
+type CoinGeckoHistoryData = {
+  prices: [number, number][]
+}
+
+type CoinGeckoMarketServiceArgs = {
+  coinGeckoAPIKey: string
+}
+
 export class CoinGeckoMarketService implements MarketService {
+  private readonly maxCount = 2500
```

Comment:
```suggestion
  private readonly defaultCount = 2500
```

this isn't/wasn't a max afaik